packages/sitecore-jss-dev-tools/src/codegen/extract-components.ts

```diff
@@ -0,0 +1,76 @@
+import { ExtractedFileType, resolveComponentImportFiles, sendCode } from './utils';
+import { fetchBearerToken } from '../auth/fetch-bearer-token';
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+type ExtractComponentArgs = {
+  appFolder?: string;
+  componentBuilderPath?: string;
+};
+
+/**
+ * Handler for the extract-component API command
+ * Reads imports from the componentBuilder.ts file and posts the code to the mesh endpoint
+ * @param {object} args - The arguments passed to the command, with optional appName string
+ * @returns {Promise<void>} void
+ */
+export async function extractComponents(args: ExtractComponentArgs) {
```

Comment:
Should be optional and in `code-extraction` plugin we don't need to pass any parameters by default
```suggestion
export async function extractComponents(args: ExtractComponentArgs = {}) {
```